contracts/System.sol

```diff
@@ -80,4 +94,104 @@ contract System {
   /// @param lowerBound requested lower bound of the param.
   /// @param upperBound requested upper bound of the param
   error OutOfBounds(string name, uint256 given, uint256 lowerBound, uint256 upperBound);
+
+  function init() external onlyNotInit {} // placeholder
+
+  function _gasprice() internal virtual view returns (uint) {
+    return tx.gasprice;
+  }
+
+  function _coinbase() internal virtual view returns (address) {
+    return block.coinbase;
+  } 
+
+  function govHub() external returns (address) {
+    return safe_govHubAddr();
+  }
+  // -------
+
+  function secureSend_(address sendTo, uint256 amount) internal returns (bool) {
```

Comment:
We need to come up with a policy for dealing with multisig players (i.e.contract not EOA) and accordingly dealing w/ reentancry attacks. 
Until then I have switched to these 'secure' functions which, at least, verifies that the balance is sufficient before transmitting while preventing reentancry attacks (unlike e.g. Address.sendValue)